lib/reservation_manager.rb

```diff
@@ -0,0 +1,113 @@
+require_relative 'room'
+
+module Hotel
+  class ReservationManager
+    
+    attr_accessor :rooms, :reservations, :blocks
```

Comment:
I'd probably use `attr_reader` here! (Your tests still pass if it's `attr_reader`)